streams/src/test/java/org/apache/kafka/streams/KafkaStreamsTest.java

```diff
@@ -90,24 +88,38 @@ public void before() {
         streams = new KafkaStreams(builder.build(), props);
     }
 
+    @After
+    public void cleanup() {
+        if (streams != null) {
+            streams.close();
```

Comment:
This method is invoked in happy path scenarios and it is also invoked when any method with `Before` is failed or test method throws an error. 
Is closing the stream here not sufficient instead of catching Exception explicitly in test methods and also closing them here?